browser-test/src/security.test.ts

```diff
@@ -4,25 +4,6 @@ import {BASE_URL} from './support/config'
 describe('applicant security', () => {
   const ctx = createTestContext()
 
-  it('applicant cannot access another applicant data', async () => {
-    const {page} = ctx
-    // this test visits page that returns 401 which triggers BrowserErrorWatcher.
-    // Silencing error on that page.
-    ctx.browserErrorWatcher.ignoreErrorsFromUrl(/applicants\/1234\/programs/)
-    const response = await gotoEndpoint(page, '/applicants/1234/programs')
-    expect(response!.status()).toBe(401)
-  })
-
-  it('admin cannot access applicant pages', async () => {
-    const {page} = ctx
-    // this test visits page that returns 401 which triggers BrowserErrorWatcher.
-    // Silencing error on that page.
-    ctx.browserErrorWatcher.ignoreErrorsFromUrl(/applicants\/1234567\/programs/)
-    await loginAsAdmin(page)
-    const response = await gotoEndpoint(page, '/applicants/1234567/programs')
-    expect(response!.status()).toBe(401)
```

Comment:
**thought:** Another option could be leaving this test and asserting a 200 response.

Reply:
I am inclined to just delete these two test cases since this is `security.test.js`, and checking for 200 doesn't really match the spirit of a security test.